### PR TITLE
Web Wallet rewording

### PR DESCRIPTION
**Pull request by BigBlueHat**
This brings the Web stuff closer to Brian's fabulous Native stuff.

### CODE REVIEW

File: developers/wallets/index.md [hunk 1]
```diff
@@ -21,37 +21,21 @@ CHAPI integrates easily into digital wallet software, allowing your wallet to re
...
 
-Wallets need to be registered with the browser as a Credential Handler in order to store or retrieve Verifiable Credentials on the Web.
+All Wallets need to be registered as a Credential Handler in the user's browser in order to store or retrieve Verifiable Credentials (VCs) from across the Web.
 
-### 1. Import the CHAPI Polyfill into your wallet app
-If you're developing on Node.js, add the credential-handler-polyfill library to your project.  You can also install the web-credential-handler helper library to simplify your code.
+A Web Wallet is typically made up of three main parts:
+* `/index.html` - main page for the wallet
+* `/manifest.json` - Web App Manifest containting credential handler affordances
+* `/wallet-worker.html` - routing endpoint for passing URLs between CHAPI and the Wallet
```
Comment (dlongley) [inline developers/wallets/index.md:34]: To make it more clear which of these specifically named relative URLs *have* to be used and which are examples:

```suggestion
* A main page for the wallet (e.g., `/index.html`)
* Web App Manifest containing credential handler affordances (using `/manifest.json`)
* A routing endpoint for handling requests between CHAPI and the Wallet (e.g., `/wallet-worker.html`) 
```

File: developers/wallets/index.md [hunk 2]
```diff
@@ -21,37 +21,21 @@ CHAPI integrates easily into digital wallet software, allowing your wallet to re
...
-console.log('Ready to work with credentials!');
-```
-<p class="code-annotation">
-  <a href="https://github.com/credential-handler/chapi-demo-wallet/blob/master/install-wallet.js"
-  target="_blank" rel="noopener noreferrer"> chapi-demo-wallet/install-wallet.js </a>
-</p>
-
-### 2. Add a `credential_handler` to your app's manifest.json
-In order to register a credential handler, your web app must serve a "manifest.json" file from its root path ("/manifest.json"). This file must also be CORS-enabled.  At a minimum, add the following `credential_handler` object:
+In order to register a credential handler, your Web app must serve a `manifest.json` file from its root path (`/manifest.json`). This file _must_ be CORS-enabled. At a minimum, the `manifest.json` file should contain the following `credential_handler` object:
```
Comment (dlongley) [inline developers/wallets/index.md:38]: We should add which CORS headers are required to help devs.

File: developers/wallets/index.md [hunk 3]
```diff
@@ -66,21 +50,66 @@ In order to register a credential handler, your web app must serve a "manifest.j
   target="_blank" rel="noopener noreferrer"> chapi-demo-wallet/manifest.json </a>
 </p>
 
-### 3. Allow your users to register their wallet's credential handler with the browser polyfill
+The `/wallet-worker.html` page will setup listeners for CHAPI-sent events
```
Comment (dlongley) [inline developers/wallets/index.md:53]:
```suggestion
The `/wallet-worker.html` page (note: you can use another name if you'd like) will setup listeners for CHAPI-sent events
```

File: developers/wallets/index.md [hunk 4]
```diff
@@ -66,21 +50,66 @@ In order to register a credential handler, your web app must serve a "manifest.j
...
 ```
-<p class="code-annotation">
-  <a href="https://github.com/credential-handler/chapi-demo-wallet/blob/master/install-wallet.js"
-  target="_blank" rel="noopener noreferrer"> chapi-demo-wallet/install-wallet.js </a>
-</p>
 
-### 4. Setup Listeners for CHAPI Events
-Configure your wallet to respond to Credential Handler events. The `activateHandler()` function is a helper that sets up listeners for CHAPI `get()` and `store()` events.
+Once you've done that and bundled it, reference the output/bundled `.js` in
+your `index.html`.
```
Comment (dlongley) [inline developers/wallets/index.md:102]:
```suggestion
your `index.html` (note: you can use another name if you'd like).
```

File: developers/wallets/native/index.md [hunk 5]
```diff
@@ -56,9 +56,14 @@ The `"acceptedInput": "url"` line tells CHAPI that the wallet should receive dat
 
 ### 2. Allow your users to register their wallet as a Credential Handler with the browser polyfill
 
-Next, we need to have the user trigger the registration of your wallet as a Credential Handler within their browser. This must be initiated by user interaction, so you cannot automatically register without the user clicking a button.
+To register with CHAPI, your user must trigger the registration of your wallet as a Credential Handler within their browser. This must be initiated by user interaction, so you cannot automatically register without the user clicking a button.
```
Comment (dlongley) [inline developers/wallets/native/index.md:59]:
```suggestion
To register with CHAPI, your user must trigger the registration of your wallet as a Credential Handler within their browser. This must be initiated by user interaction, so you cannot automatically register without the user clicking or tapping a button or link.
```

**Review by dlongley — APPROVED** [review]
Thanks! Approving with some suggestions.

File: developers/wallets/index.md (hunk 1 above)
Comment (BigBlueHat) [reply developers/wallets/index.md:34]: I rather liked the filename first approach as it gave me ready handles to hook onto later, but happy to swap them. I'll muse on it a bit.

File: developers/wallets/index.md (hunk 2 above)
Comment (BigBlueHat) [reply developers/wallets/index.md:38]: @dlongley are their suggested ones? Likely don't want everyone taking the "easy" road and just `Access-Control-Allow-Origin: *` ...but we also don't want them having to chase the domain name around--should it change.

@brianorwhatever you've done this recently. Any suggestions here on CORS specifics?

File: developers/wallets/index.md (hunk 3 above)
Comment (BigBlueHat) [reply developers/wallets/index.md:53]: I did have something like that at one point...I'll put it back.

File: developers/wallets/index.md (hunk 1 above)
Comment (BigBlueHat) [reply developers/wallets/index.md:34]: k...you talked me into it... 😉

File: developers/wallets/index.md (hunk 2 above)
Comment (brianorwhatever) [reply developers/wallets/index.md:38]: * is required as it could be any domain on the web that needs to pull this file down

**Pull request merged**